teamcity-export/src/main/kotlin/org/gradle/devprod/collector/teamcity/TeamcityExport.kt

```diff
@@ -12,66 +8,60 @@ import java.time.temporal.ChronoUnit
 
 @Component
 class TeamcityExport(
-    private val create: DSLContext,
+    private val repo: Repository,
     private val teamcityClientService: TeamcityClientService
 ) {
+
+    private val gbtPipelines = listOf("Master", "Release")
+    private val gbtBuildConfigurations: (String) -> List<String> = {
+        pipeline: String ->
+        listOf(
+            "Gradle_${pipeline}_Check_Stage_QuickFeedbackLinuxOnly_Trigger",
+            "Gradle_${pipeline}_Check_Stage_QuickFeedback_Trigger",
+            "Gradle_${pipeline}_Check_Stage_PullRequestFeedback_Trigger",
+            "Gradle_${pipeline}_Check_Stage_ReadyforNightly_Trigger",
+            "Gradle_${pipeline}_Check_Stage_ReadyforRelease_Trigger"
+        )
+    }
+    private val gbtRootProjectAffectedBuild: (String) -> String = {
+        pipeline: String ->
+        "Gradle_${pipeline}_Check"
+    }
```

Comment:
```suggestion
    private val gbtRootProjectAffectedBuild: (String) -> String = { "Gradle_${it}_Check" }
```

This could be an oneliner. If you prefer explicit parameter, I think the common convention is to put it at the same line as left bracket, i.e.

```
    private val gbtRootProjectAffectedBuild: (String) -> String = { pipeline: String ->
        "Gradle_${pipeline}_Check"
    }
```

Reply:
done